### ISSUE TITLE
Links in plugin explorer are missing underline

### ISSUE THREAD

**Issue by kevee** [gatsbyjs/gatsby#30394]
## Description

Links in the content area of the [plugins section](https://www.gatsbyjs.com/plugins) of the website are difficult to discern, and fail WCAG SC 1.4.1.

### Steps to reproduce

- Look up any plugin in the plugin section, for example [gatsby-image](https://www.gatsbyjs.com/plugins/gatsby-image/).
- Look at the main content of the page from the plugin readme

### Expected result

The links in the content area should have an underline, like the rest of the site. For example:
![image](https://user-images.githubusercontent.com/270536/112029968-00359980-8af7-11eb-8ed8-db658d8d6306.png)

### Actual result

The links are differentiated by a dark purple color only, without an underline, making them very difficult to discern from normal text:
![image](https://user-images.githubusercontent.com/270536/112030070-1cd1d180-8af7-11eb-8d36-00ec9548c47a.png)

### Environment

N/A

**Comment by LekoArts** [comment]
Thanks for the issue! I filed this internally and we'll get it fixed.
